server/controllers/controller.js

```diff
@@ -1,6 +1,41 @@
 const Model = require('../db-models/database');
 const axios = require('axios');
 
+exports.createGallery = async ({ headerImage, mainImages, moreLikeThisImages }) => {
+  let count = await Model.countDocuments({}, function (err, count) {
+    if (err) {
+        console.log(err);
+    } else {
+        return count++;
+    }
+  });
+
```

Comment:
I'm less clear about how mongodb error handling works with async/await but I think you _might_ be able to handle it in true async/await form:

`
try {
  let count = await Model.countDocuments({})
  return count++;
} catch(err) {
  console.log(err);
}`

Reply:
this felt really clunky when i wrote it. i like the try/catch syntax and saw some good examples in your code.